### DISCUSSION TITLE
Style a header

### DISCUSSION THREAD

**Discussion by ichuprov** [filamentphp/filament#8917] [General]
Perhaps I am not searching the right thing, but I can't seem to find a simple example of how to change the style of a form field heading. All I'd like to do is make the form field name look bolder so it looks different to what is showing in a preview form.
Just a quick example here: I just want to style First Name to appear bolder, thats it
![image](https://github.com/filamentphp/filament/assets/1366482/e5228969-92b4-48d8-9a9c-9dcbf2b73f04)

**Comment by leandrocfe** [comment]
Maybe you can [create a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) and edit via css
